lists/global.csv

```diff
@@ -1234,3 +1234,16 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+https://doubleclick.net/,COMM,E-commerce,2017-03-02,CIPIT,
+http://googleusercontent.com/,SRCH,Search Engines,2017-03-02,CIPIT,
```

Comment:
I don't see any content at this site.

Reply:
Ah! I think it changed to:
http://dl.googleusercontent.com/

Reply:
Should I remove it or change?

Reply:
I get a 404 when accessing dl.googleusercontent.com. I see that several sub-domains of googleusercontent.com are used for various purposes, but can't seem to find the most canonical one.

Maybe we should drop it unless we have a specific subdomain that is deemed important to test?

Reply:
it's reasonable.